server/gateway/src/app.ts

```diff
@@ -143,14 +147,13 @@ export function create(
     // Static cache to help map from full to minified files
     const staticMinCache: { [key: string]: string } = {};
 
-    const microsoftConfiguration = config.get("login:microsoft");
     passport.use(
         new passportOpenIdConnect.Strategy(
             {
                 authorizationURL: "https://login.microsoftonline.com/organizations/oauth2/v2.0/authorize",
                 callbackURL: "/auth/callback",
-                clientID: microsoftConfiguration.clientId,
-                clientSecret: microsoftConfiguration.secret,
+                clientID: process.env.MICROSOFT_CONFIGURATION_CLIENT_ID,
+                clientSecret: process.env.MICROSOFT_CONFIGURATION_CLIENT_SECRET,
```

Comment:
Does this impact the existing gateway deployments in any way? Do you need any updates there before checking in?

Reply:
This will not work in the cluster. I spoke to @sumedhb1995 about alternate options. Basically you have to consider both something clientID: config.get("login:microsoft:clientid") || process.env.MICROSOFT_CONFIGURATION_CLIENT_ID

Otherwise the cluster does not know how to get those values. I am sure there is an way to let the cluster know as well but I have never done that.

Reply:
I've switched it back to only use the local .env values when the config values are not available.
This should prevent any accidental checkins of client secrets while also preserving our release process

Reply:
Also restored the empty config.json to have places for the release step to still fill in the secrets